src/VscProcess.cpp

```diff
@@ -288,25 +288,50 @@ int VscProcess::handleHeartbeatMsg(VscMsgType& recvMsg)
     estopValue.data = msgPtr->EStopStatus;
     estopPub.publish(estopValue);
 
+    // Static values allow us to detect when the state changes
+    static bool prev_estop_vehicle = false;
+    static bool prev_estop_src = false;
+    static bool prev_estop_any = false;
```

Comment:
This does not get set anywhere else.

Reply:
Thanks! I just fixed it. I added the `any` one later than the first two.